dev/core-bundle/serve-env.json

```diff
@@ -1,6 +1,8 @@
 {
   "HELMREVISION": "dev",
   "NETWORK": "regtest",
+  "BYPASS_ROLE_CHECK": "true",
+  "NODE_ENV": "test",
```

Comment:
is this not used in dev too? this value should only be used for gh actions and ci